api/crypto/frame_crypto_transformer.cc

```diff
@@ -490,25 +508,22 @@ void FrameCryptorTransformer::decryptFrame(
   rtc::ArrayView<const uint8_t> date_in = frame->GetData();
 
   if (date_in.size() == 0 || !enabled_cryption) {
-    sink_callback->OnTransformedFrame(std::move(frame));
+    RTC_LOG(LS_WARNING)
+      << "FrameCryptorTransformer::decryptFrame() date_in.size() == 0 || enabled_cryption == false";
     return;
   }
 
   auto uncrypted_magic_bytes = key_provider_->options().uncrypted_magic_bytes;
   if (uncrypted_magic_bytes.size() > 0 &&
-      date_in.size() >= uncrypted_magic_bytes.size() + 1) {
+      date_in.size() >= uncrypted_magic_bytes.size()) {
     auto tmp =
-        date_in.subview(date_in.size() - (uncrypted_magic_bytes.size() + 1),
```

Comment:
is it the only change made to sif?

Reply:
Yes, when we discussed with @lukasIO  earlier, we reserved a 1 byte for frameType. It seems that it has been abandoned in the latest js-sdk.

Reply:
yeah sorry, we decided on that when discussing with Raja, should have made this more explicit!